R/load.R

```diff
@@ -146,22 +146,22 @@ renv_load_action <- function(project) {
   if (!interactive())
     return("load")
 
-  # if this project already contains an 'renv' folder, assume it's
-  # already been initialized and we can directly load it
-  renv <- renv_paths_renv(project = project, profile = FALSE)
-  if (dir.exists(renv))
-    return("load")
-
   # if we're running within RStudio at this point, and we're running
   # within the auto-loader, we need to defer execution here so that
   # the console is able to properly receive user input and update
   # https://github.com/rstudio/renv/issues/1650
   autoloading <- getOption("renv.autoloader.running", default = FALSE)
   if (autoloading && renv_rstudio_available()) {
-    setHook("rstudio.sessionInit", function() { renv::load(project) })
+    setHook("rstudio.sessionInit", function(...) { renv::load(project) })
     return("cancel")
```

Comment:
Morally, I want to do `quietly(return("cancel"))`, but that doesn't actually work.

Reply:
Handles by adding a `verbose` argument to `cancel()`.